expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -307,6 +365,12 @@ private static void addPublisherToMockUriRetriever(UriRetriever uriRetriever, UR
         mockGetRawContentResponse(uriRetriever, publisherId, publicationChannelSamplePublisher);
     }
 
+//    private static void addFundingsToMockUriRetriever(UriRetriever uriRetriever, URI seriesId) throws IOException {
```

Comment:
Is there a reason you have left the commented code here?

Reply:
poor eye sight :)